packages/components/src/FormatFile/FormatFile.css

```diff
@@ -137,13 +137,15 @@
   right: var(--space-smaller);
 }
 
-.deleteable ~ .deleteButton {
-  visibility: hidden;
-}
-
-.deleteButton:focus-within,
-.deleteButton:hover,
-.deleteable:hover ~ .deleteButton,
-.deleteable:focus ~ .deleteButton {
-  visibility: visible;
+@media (--medium-screens-and-up) {
+  .deleteable ~ .deleteButton {
+    visibility: hidden;
+  }
+
+  .deleteButton:focus-within,
```

Comment:
Moved this into the media query because it will default to be visible on small breakpoints, as a result we only need to do the `hover` updates when on non-mobile breakpoints